cmd/downloads/config/artifactsFileSpec.yaml

```diff
@@ -0,0 +1,22 @@
+artifacts:
```

Comment:
Lets rename the file to something like `defaultArtifactsConfig`, since in production we will be overriding the config with CM, placed in our operator , which will be mounted into the `downloads` pod.